VMBackup/main/Utils/HandlerUtil.py

```diff
@@ -116,7 +116,11 @@ def __init__(self, log, error, short_name):
 
     def _get_log_prefix(self):
         return '[%s-%s]' % (self._context._name, self._context._version)
-
+    
+    # It is possible in certain restore scenarios, where old setting files are present in the config folder
+    # When new VM is created with same extension, the seq# starts with 0 again. 
```

Comment:
probably better to explain how older setting files are retained in a new VM? I guess you are referring to the case when OS disk is created from a restore point or a snaphost.